gravitee-apim-portal-webui-next/src/app/api-details/api-tab-subscriptions/api-tab-subscriptions.component.scss

```diff
@@ -13,10 +13,23 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
+@use '../../../scss/theme';
+
 .api-tab-subscription__empty {
   display: flex;
   flex-direction: column;
   align-items: center;
   justify-content: center;
   padding: 20px 0;
 }
+
+.api-tab-subscriptions__table {
+  overflow: hidden;
+  border: 1px solid color-mix(in srgb, theme.$secondary-main-color 35%, transparent);
+  border-radius: 16px;
+
+  &-row {
+    background: #fff;
```

Comment:
Instead of using hex-code, use a variable. You can reference the mat card background color so that it matches. If we ever change to dark-mode, it will follow the same choice as the mat-cards.

Reply:
Done